src/pages/SignUp/SignUp.js

```diff
@@ -1,9 +1,341 @@
 import React, { Component } from 'react';
+import Footer from '../../components/Footer/Footer';
 import './SignUp.scss';
 
 class SignUp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      userRulePlusClassName: 'userRulePlusOff',
+      userInfoPlusClassName: 'userInfoPlusOff',
+      checked: false,
```

Comment:
```suggestion
      isChecked: false,
```
boolean 값을 표현하는 경우 `is-` 접두사를 붙여주면 데이터 형태를 쉽게 추론할 수 있을 것 같습니다

Reply:
넵 ~~!